nixos/modules/services/home-automation/zigbee2mqtt.nix

```
@@ -76,9 +76,7 @@ in
 
         # Hardening
         CapabilityBoundingSet = "";
-        DeviceAllow = [
-          config.services.zigbee2mqtt.settings.serial.port
-        ];
+        DeviceAllow = lib.optionals (lib.hasPrefix "/dev/" cfg.settings.serial.port) [ cfg.settings.serial.port ];
```

```suggestion
        DeviceAllow = lib.optionals (lib.hasPrefix "/" cfg.settings.serial.port) [ cfg.settings.serial.port ];
```

I think we could go even that far, to allow any absolute path. URIs surely won't start with a slash.